.gitignore

```diff
@@ -130,3 +130,5 @@ reports/
 .DS_Store
 
 target/
+
+.bash_profile
```

Comment:
Shouldn't bash_profile be on your root anyway?

Reply:
I tried putting it into the root, but the settings won't cascade to the actual arrow project. Can you help me, maybe I did it wrong. I have a Macbook Pro 10.14.3

Reply:
You may have needed to restart your terminal, or source the bash profile? I use zsh.